vars/user.yml

```diff
@@ -87,3 +87,6 @@ selfoss_password_hash: "TODO"
 # wallabag
 wallabag_salt: TODO
 wallabag_db_password: TODO
+
+mail_db_opendmarc_username: opendmarc
```

Comment:
This username is already in ``defaults.yml``. Is it expected that the user will change it?